backend/src/main/kotlin/app/ehrenamtskarte/backend/mail/EmailBody.kt

```diff
@@ -63,6 +67,17 @@ class Paragraph {
         children.add(Text(this))
     }
 
+    fun plain(plainText: String) {
+        val sentences = plainText.split(keepNewLine)
+        sentences.forEach {
+            if (it.matches(newLine)) {
+                children.add(LineBreak())
+            } else {
+                children.add(Text(it))
+            }
+        }
```

Comment:
I think, working with these complicated regexes is not necessary 🤔 

```suggestion
        plainText.trim().split("\n").forEachIndexed { index, text ->
            if (index != 0)
                children.add(LineBreak())
            children.add(Text(text))
        }
```